lighthouse-core/report/html/renderer/util.js

```diff
@@ -26,6 +26,40 @@ class Util {
     return PASS_THRESHOLD;
   }
 
+  static get MS_DISPLAY_VALUE() {
+    return `%10d${NBSP}ms`;
+  }
+
+  /**
+   * @param {string|Array<string|number>=} displayValue
+   * @return {string}
+   */
+  static formatDisplayValue(displayValue) {
+    if (typeof displayValue === 'string' || typeof displayValue === 'undefined') {
+      return displayValue || '';
+    }
+
+    const template = /** @type {string} */ displayValue.shift();
+    if (typeof template !== 'string') {
+      // First value should always be the format string, but we don't want to fail to build
+      // a report, return a placeholder.
+      return 'UNKNOWN';
+    }
+
+    let output = template;
+    while (displayValue.length) {
+      const replacement = /** @type {number|string} */ (displayValue.shift());
+      output = output.replace(/%([0-9.]+)?(d|s)/, match => {
+        const granularity = Number(match.match(/[0-9.]+/)) || 1;
+        return match === '%s' ?
```

Comment:
```js
assert.equal(format(['%10s', 123.456]), '120');
```

currently this _does_ pass, fwiw. shoudl we also throw a warning if you are matching 's' and have a granularity?